apps/tb_meetings/src/tb_meetings_server.erl

```diff
@@ -0,0 +1,61 @@
+-module(tb_meetings_server).
+-behaviour(gen_server).
+-behaviour(deliverly_handler).
+-define(SERVER, ?MODULE).
+
+%% API Function Exports
+-export([start_link/0]).
+
+%% gen_server Function Exports
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2,
+         terminate/2, code_change/3]).
+
+%%% Deverly Handler callbacks.
+-export([authorize/2, handle_message/2, handle_client_message/2, client_disconnected/1]).
```

Comment:
Line 14 is too long: -export([authorize/2, handle_message/2, handle_client_message/2, client_disconnected/1])..